omod/src/main/web/dashboardwidgets/dataintegrityviolations/index.js

```diff
@@ -1,6 +1,6 @@
 import angular from 'angular';
 import openmrsApi from '@openmrs/angularjs-openmrs-api';
-import { DatatIntegrityViolationsComponent } from './dataintegrityviolations.component';
+import { DataIntegrityViolationsComponent } from './dataintegrityviolations.component';
 
 export default angular.module("openmrs-contrib-dashboardwidgets.dataintegrityviolations", [ openmrsApi ])
-	.component(DatatIntegrityViolationsComponent.selector, DatatIntegrityViolationsComponent).name;
\ No newline at end of file
+	.component(DataIntegrityViolationsComponent.selector, DataIntegrityViolationsComponent).name;
```

Comment:
@shivtej1505 okay - I would suggest correcting the name after making the changes we need so that you have isolated changes and can know what is breaking. Too many changes mean that you will spend lots of time troubleshooting . There are two implementations that need this ASAP